Workbooks/Workloads/Collection logs/settings.json

```diff
@@ -2,7 +2,6 @@
     "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/settings.json",
     "name":"View collection troubleshooting logs",
     "author": "Microsoft",
-    "galleries": [
-        { "type": "workloads-shared", "resourceType": "Azure Monitor", "order": 500 }
-    ]
+    "isPreview": true,

```

Comment:
should isPreview be false now?

Reply:
doens't need to be since it won't show, I'll leave as is for now